Detectors/TPC/qc/src/Clusters.cxx

```diff
@@ -79,13 +89,46 @@ void Clusters::fillADCValue(int cru, int rowInSector, int padInRow, int timeBin,
 }
 
 //______________________________________________________________________________
-void Clusters::analyse()
+void Clusters::normalize()
 {
+  if (mIsNormalized) {
+    return;
+  }
+
   mQMax /= mNClusters;
   mQTot /= mNClusters;
   mSigmaTime /= mNClusters;
   mSigmaPad /= mNClusters;
   mTimeBin /= mNClusters;
+
+  mIsNormalized = true;
+}
+
+//______________________________________________________________________________
+void Clusters::denormalize()
+{
+  if (!mIsNormalized) {
+    return;
+  }
+
+  mQMax *= mNClusters;
+  mQTot *= mNClusters;
+  mSigmaTime *= mNClusters;
+  mSigmaPad *= mNClusters;
+  mTimeBin *= mNClusters;
+
+  mIsNormalized = false;
+}
+
+//______________________________________________________________________________
+void Clusters::reset()
+{
+  mNClusters *= 0;
+  mQMax *= 0;
+  mQTot *= 0;
+  mSigmaTime *= 0;
+  mSigmaPad *= 0;
+  mTimeBin *= 0;
```

Comment:
I don't know how often this method is used, but what is the reason of multiplying by 0 instead of assignment?

Reply:
Hi @shahor02, these member variables are CalDet<float> objects. Assigning 0 would not work.